script/js_7/index.js

```diff
@@ -0,0 +1,89 @@
+//1.1
+var playList = [
+    {
+      author: "LED ZEPPELIN",
+      song: "STAIRWAY TO HEAVEN"
+    },
+    {
+      author: "QUEEN",
+      song: "BOHEMIAN RHAPSODY"
+    },
+    {
+      author: "LYNYRD SKYNYRD",
+      song: "FREE BIRD"
+    },
+    {
+      author: "DEEP PURPLE",
+      song: "SMOKE ON THE WATER"
+    },
+    {
+      author: "JIMI HENDRIX",
+      song: "ALL ALONG THE WATCHTOWER"
+    },
+    {
+      author: "AC/DC",
+      song: "BACK IN BLACK"
+    },
+    {
+      author: "QUEEN",
+      song: "WE WILL ROCK YOU"
+    },
+    {
+      author: "METALLICA",
+      song: "ENTER SANDMAN"
+    }
+  ];
+  const ol = document.createElement("ol");
+  const mainDiv = document.getElementById("content");
+  mainDiv.append(ol);
+  for (let i in playList) {
+    const li = document.createElement("li");
+    const pBtn = document.createElement("p");
+    pBtn.append(playList[i].author);
```

Comment:
You can do `pBtn.append(`$playList[i].author} : ${playList[i].song});`

It will save you more lines